glide.yaml

```diff
@@ -1,13 +1,15 @@
 package: github.com/DataDog/datadog-process-agent
 import:
   - package: github.com/DataDog/datadog-agent
-    version: f1d63c176a934c8b8aefa31e2c7d938b0e50e348
+    vcs: git
+    version: db/merge_no_docker
```

Comment:
Does it still need to be pinned to this branch, or can we change it to the latest `master`?